KabernyukIS/lab_1/main.cpp

```diff
@@ -0,0 +1,76 @@
+#include <iostream>
+#include <string>
+
+
+int sumAscii(std::string a) {
+	int count = 0;
+	for (int i = 0; i < a.size(); i++) {
+		if ((int)a[i] >= 65) {
+			count += (int)a[i];
+		}
+		else {
+			count = 0;
+			break;
+		}
+	}
+	return count;
+}
+//Функция подсчета суммы ASCII символов
+
+int main() {
+
+	std::cout << "Enter your string: ";
+	std::string text;
+	std::getline(std::cin, text);
+	int k = 0, i = 0, j;
+	std::string tmp, words[100];
+	//Объявление переменных
+
+	for (int i = 1; i < text.size(); i++) {
+		if (((int)text[i] < 65) && ((int)text[i] != 32) && (((int)text[i - 1] >= 65 && (int)text[i + 1] >= 65) || ((int)text[i - 1] >= 65 && (int)text[i + 1] < 65 && (int)text[i + 1] > 32) || ((int)text[i - 1] < 65 && (int)text[i + 1] >= 65 && (int)text[i - 1] > 32))) {
```

Comment:
Please replace numeric literals with corresponding char literals e.g. text[i] < 'A'
Also cast to int is redundant